src/bun.js/bindings/napi.cpp

```diff
@@ -967,6 +967,21 @@ extern "C" napi_status napi_get_and_clear_last_exception(napi_env env,
     return napi_ok;
 }
 
+extern "C" napi_status napi_fatal_exception(napi_env env,
+    napi_value err)
+{
+    auto globalObject = toJS(env);
+    JSC::JSValue value = JSC::JSValue::decode(reinterpret_cast<JSC::EncodedJSValue>(err));
+    JSC::JSObject* obj = value.getObject();
+    if (UNLIKELY(obj == nullptr || !obj->isErrorInstance())) {
```

Comment:
There is a subtle difference between an exception and an error and I'm not sure this satisfies that

Reply:
But I'm also not sure if the value passed here is expected to be a JSC::Exception

Reply:
I think we can merge this since its better than the present state regardless